cg/cli/store.py

```diff
@@ -77,7 +77,10 @@ def _include_files_in_housekeeper(bundle_obj, context, hk_api, version_obj):
 def _add_new_complete_analysis_record(bundle_data, family_obj, status, version_obj):
 
     pipeline = family_obj.links[0].sample.data_analysis
-    pipeline = pipeline if pipeline else 'mip' # TODO remove this default from here
+
+    if status.analysis(family=family_obj, started_at=version_obj.created_at):
```

Comment:
haha, not assigning it was the solution to the faulty query? :D

Reply:
Well, the faulty query was due to uninitiated mock object